data/access.log

```diff
@@ -0,0 +1,4 @@
+tp 4
```

Comment:
This test case was just to test that part 1 and 2 worked. I can write a more general one